docs/UserGuide.md

```diff
@@ -309,7 +309,7 @@ Format: `add -l n/LESSON_NAME [c/LESSON_CAPACITY] [p/LESSON_PRICE] [t/LESSON_TIM
 * If provided, the lesson's price must be a **non-negative number** with either 0 or 2 decimal places. Examples of a valid price are `80` and `85.50`.
 
 Examples:
-* `add -l n/P6 Maths c/20 p/80 t/Monday 1200-1400`
+* `add -l n/P6 Maths c/20 p/80 t/Monday 1200-1400` adds a lesson with name P6 Maths, capacity 20, price 80 and timing Monday 1200-1400.
```

Comment:
Maybe it might be more complete to write it as such: `adds a lesson with the name 'P6 Maths', capacity of 20 students, price of $80 and the timing 'Monday 1200-1400'`